libs/libc/signal/sig_ismember.c

```diff
@@ -63,17 +66,53 @@
  *
  ****************************************************************************/
 
-int sigismember(FAR const sigset_t *set, int signo)
+int nxsig_ismember(FAR const sigset_t *set, int signo)
 {
-  int ret = ERROR;
-
   /* Verify the signal */
 
-  if (GOOD_SIGNO(signo))
+  if (!GOOD_SIGNO(signo))
+    {
+      return ERROR;
```

Comment:
Similar to the above. ERROR should never be returned inside the OS. It should return -EINVAL if the signal number is invalid.

Reply:
Done.